README.md

```diff
@@ -87,7 +87,12 @@ export default new OAuthProvider({
   // Note: Creating public clients via the OAuthHelpers.createClient() method
   // is always allowed regardless of this setting.
   // Defaults to false.
-  disallowPublicClientRegistration: false
+  disallowPublicClientRegistration: false,
+
+  // Optional: Time-to-live for refresh tokens in seconds.
```

Comment:
We should also have an option for "no refresh token" (only access token). Maybe set this to zero?